draft-ietf-ppm-dap.md

```diff
@@ -2654,6 +2654,21 @@ the input shares and defeat privacy.
 
 # IANA Considerations
 
+## Message versioning
```

Comment:
Let's move this to a subsection of `## Protocol Message Media Types` immediately below this.

Reply:
Moved.